server/src/messageHandler.mjs

```diff
@@ -47,6 +48,31 @@ async function handleMessage(message, userId, ws) {
     return;
   }
 
+  //bypass JSON-RPC calls and directly target device. 
+  if(process.env.proxy) {
+    let params = oMsg.params
+    const payload = {
+      method: oMsg.method,
+      jsonrpc: "2.0",
+      id: oMsg.id
+    }
+    if(params) {
+      payload.params = params
+    }
+    
+    let wsProxy = await proxyManagement.getProxyWSConnection()
+    if( ! wsProxy ) {
+      //init websocket connection for proxy request to be sent and update receiver client to send request back to caller.
+      wsProxy = await proxyManagement.initialize(ws)
```

Comment:
What if initialization fails? We should catch the error and fail gracefully here.

Reply:
Addressed